src/FluentValidation/AbstractValidator.cs

```diff
@@ -236,9 +236,11 @@ ValueTask<ValidationResult> completedValueTask
 		EnsureInstanceNotNull(context.InstanceToValidate);
 #pragma warning restore CS0618
 
-		foreach (var rule in Rules) {

```

Comment:
Can you add a comment here indicating that this was switched from a `foreach` to a `for` for perf reasons so future contributors don't try and change it back.

Also please could you fix the indentation so its consistent with the rest of the file?

Reply:
Resolved in 4423adcbbde4d5574136e69d468676d0eaa8709d